joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

position_joint_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7
  constraints:
    goal_time: 0.5
    stopped_velocity_tolerance: 0.05
    panda_joint1: {trajectory: 0.1, goal: 0.1}
    panda_joint2: {trajectory: 0.1, goal: 0.1}
    panda_joint3: {trajectory: 0.1, goal: 0.1}
    panda_joint4: {trajectory: 0.1, goal: 0.1}
    panda_joint5: {trajectory: 0.1, goal: 0.1}
    panda_joint6: {trajectory: 0.1, goal: 0.1}
    panda_joint7: {trajectory: 0.1, goal: 0.1}
  gains:
    panda_joint1: {p: 100, d: 1, i: 1}
    panda_joint2: {p: 100, d: 1, i: 1}
    panda_joint3: {p: 100, d: 1, i: 1}
    panda_joint4: {p: 100, d: 1, i: 1}
    panda_joint5: {p: 100, d: 1, i: 1}
    panda_joint6: {p: 100, d: 1, i: 1}
    panda_joint7: {p: 100, d: 1, i: 1}

